src/components/textarea/textarea.stories.mdx

```diff
@@ -102,6 +102,17 @@ import Textarea from "carbon-react/lib/components/textarea";
   />
 </Canvas>
 
+### With custom maxWidth
+
+<Canvas>
+  <Story name="with custom maxWidth">
```

Comment:
could you move this story to the `.stories.tsx` file and add the imported name to the `story` prop of the `<Story>` component as done in the others? Just for consistency and because this is the way we've agreed to do it.